admin/src/app/home/orders/stock-orders/generated/generated.component.ts

```diff
@@ -518,8 +580,20 @@ export class GeneratedComponent implements OnInit, OnDestroy {
     }
     this.orgModelApi.updateAllStockOrderLineItemModels(this.userProfile.orgModelId, this.order.id, lineItemsIDs, data)
       .subscribe((res: any) => {
-          this.reloadData();
-        },
+        if (lineItems) {
```

Comment:
what happens if the updateAPI fails? shouldn't the item move back to its original list with an error displayed?

Reply:
If the Api fails teh item wont move to the new list